language/move-command-line-common/src/values.rs

```diff
@@ -41,10 +41,13 @@ pub enum ValueToken {
 #[derive(Eq, PartialEq, Debug, Clone)]
 pub enum ParsedValue<Extra: ParsableValue = ()> {
     Address(ParsedAddress),
-    InferredNum(u128),
+    InferredNum(move_core_types::u256::U256),
```

Comment:
Same here, maybe just import U256?

But you can do this in a future PR :)

Reply:
Yup!